docs/getting_started/quick_start.rst

```diff
@@ -157,6 +157,16 @@ to read-only mode whenever you want:
     If you are uncomfortable hard-coding your credentials into your program, there are
     some options available to you. Please see: :ref:`configuration`.
 
+Close Connections in :class:`.Reddit`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The asynchronous context requires closing your session when you finish using Reddit:
+
+.. code-block:: python
+
+    reddit = asyncpraw.Reddit(...)
+    await reddit.close()
+
```

Comment:
Could you add using `Reddit` as an asynchronous context manager? I think there is an example in the docs for the Reddit class. If not, here's this:

```
async with asyncpraw.Reddit(...) as reddit:
    print(await reddit.user.me()
```